container/pcct/README.md

```diff
@@ -3,41 +3,41 @@
 This package implements the **PIT-CS Composite Table (PCCT)**, a non-thread-safe hash table that carries both the Pending Interest Table (PIT) and the Content Store (CS).
 
 Each **PCC entry** contains PIT and CS entries for a combination of Interest/Data name and chosen delegation name in the forwarding hint.
-It can store a PIT entry for MustBeFresh=0 (denoted "PitEntry0"), a PIT entry for MustBeFresh=1 (denoted "PitEntry1"), and a CS entry.
+It can store a PIT entry for *MustBeFresh=0* (denoted "PitEntry0"), a PIT entry for *MustBeFresh=1* (denoted "PitEntry1"), and a CS entry.
 The main `PccEntry` type has one `PccSlot` providing room for one PIT entry or one CS entry.
-An `PccEntryExt` can be allocated from the PCCT's mempool to provide two additional slots.
+A `PccEntryExt` can be allocated from the PCCT's mempool to provide two additional slots.
 Regardless of available slots, each PCC entry can only have one each of PitEntry0, PitEntry1, and CS entry.
 
-C code for [PIT](../pit/) and [CS](../cs/) is in this directory to avoid circular dependency problems, but their Go bindings and documentation are in their own packages.
+The C code for the [PIT](../pit) and the [CS](../cs) belongs to this package to avoid circular dependencies, but their respective Go bindings and related documentation are in their own separate packages.
```

Comment:
This is no longer accurate.
C code is in csrc/pcct. They are still in the same directory to avoid circular dependency between directories.

Reply:
right, the C code is no longer in this directory but it's still logically part of this package, no? I changed "directory" to "package"

Reply:
Also, since all C code is now compiled into a static library, where do you get circular dependencies anyway?

Reply:
PCCT depends on PIT, and PIT depends on PCCT. However, two headers cannot `#include` each other.